src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -39,11 +62,8 @@ const ActorInit En_Rz_InitVars = {
     (ActorFunc)EnRz_Draw,
 };
 
-TexturePtr D_80BFCCE0[] = { object_rz_Tex_00BC50, object_rz_Tex_00C190, object_rz_Tex_00C590 };
-
-TexturePtr D_80BFCCEC = object_rz_Tex_00C990;
-
-TexturePtr D_80BFCCF0 = object_rz_Tex_00CD90;
+TexturePtr D_80BFCCE0[] = { object_rz_Tex_00BC50, object_rz_Tex_00C190, object_rz_Tex_00C590, object_rz_Tex_00C990,
+                            object_rz_Tex_00CD90 };
```

Comment:
```suggestion
TexturePtr sEyeTextures[] = { object_rz_Tex_00BC50, object_rz_Tex_00C190, object_rz_Tex_00C590, object_rz_Tex_00C990,
                            object_rz_Tex_00CD90, };
```
reformat, and might as well make it static.